bundler/lib/dependabot/bundler/helpers.rb

```diff
@@ -32,9 +30,7 @@ def self.bundler_version(lockfile)
       # it was created with an old version that didn't add this information
```

Comment:
This code comment says "we failover to v1"... but that's no longer true...

Reply:
fixed.